src/test/scala/chiseltest/tests/OptionsPassingTest.scala

```diff
@@ -6,10 +6,12 @@ import chisel3._
 import chiseltest._
 import chiseltest.experimental.TestOptionBuilder._
 import chiseltest.experimental.sanitizeFileName
+import firrtl.stage.OutputFileAnnotation
+import logger.{LazyLogging, Logger}
 import org.scalatest._
 import treadle.{VerboseAnnotation, WriteVcdAnnotation}
 
-class OptionsPassingTest extends FlatSpec with ChiselScalatestTester with Matchers {
+class OptionsPassingTest extends FlatSpec with ChiselScalatestTester with Matchers with LazyLogging {
```

Comment:
Whats this LazyLogging for?

Reply:
Leftover from an earlier test attempt. Removed